.gitignore

```diff
@@ -28,3 +28,6 @@ profile.out
 
 simplest-uncommitted-msg-0.1-jar-with-dependencies.jar
 
+.idea
```

Comment:
This can be added in your global gitignore, this is very specific to your dev workflow

Reply:
yeah, I PR'd from a machine that doesn't have all the settings. will fix it now.